### ISSUE TITLE
Problem while trying to convert AA to 3Diaa

### ISSUE THREAD

**Issue by mauriciolangleib** [mheinzinger/ProstT5#21]
Hi! First of all many thanks for developing such a useful tool.

I am running into the following problem while executing the code provided in this repository to convert AA to 3Diaa (I paste a screenshot below).

![image](https://github.com/mheinzinger/ProstT5/assets/30030684/2d1149bc-7502-4203-a1c7-c5ca02097b56)

Due to my GPU, which is compatible with CUDA 11.5, I am trying to run ProstT5 with PyTorch 1.11.0, is this possible or the error might be related to this?

Many thanks in advance and sorry if I am being not very specific in my answer, I am new to this field.

**Comment by mheinzinger** [comment]
Hi! - thanks a lot for your interest in our tool :) 
On your error: I do not think that its related to the CUDA/Pytorch version you are using as the error sounds more fundamental (wrong datatype of some tensors during generation - probably either input_ids or attention_mask?).
Could you cross-check whether you might simply use the CLI script that we provide to go from AA to 3Di (called  predict_3Di_encoderOnly.py here: https://github.com/mheinzinger/ProstT5/tree/main/scripts)?  
This should work (no datatype issues expected), so it might minimally help debugging, or, depending on your use-case solve the issue.

**Comment by mauriciolangleib** [comment]
Using that script solved the issue quite fast! I had tried with the one named translate.py and it had crashed, but this one worked smoothly.

Again, many thanks for this tool, I used it to translate a huge amount of sequences (~200000) and it did it in quite an impressive time :D. Really looking forward to see this published and its evolution, seems to be a magnificent tool.

Cheers,
Mauricio
